docs/authentication/configuration/restrictions.mdx

```diff
@@ -1,91 +1,79 @@
-
-<Images
-width={3024}
-height={1654}
-src="/docs/images/authentication/restrictions/blocklist.jpg"
-alt="The 'Blocklist' section in the 'Restrictions' tab under the 'User & Authentication' section of the Clerk Dashboard. A red arrow is pointing to the 'Enable blocklist' toggle, which is toggled on."
-/>
-
-After turning on the Blocklist feature, you can add individual email addresses and phone numbers, or blocklist entire email domains.
-
-For example, if you add "clerk.dev" as a blocked email domain, it means that anybody with a "@clerk.dev" email address will not be able to sign up for your application. Email addresses from different domains will not be affected.
+To enable this feature:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)**.
+3. In the **Blocklist** section, toggle on **Enable blocklist**.
 
 <Callout type="warning" emoji="⚠️">
   In the case you have enabled the Allowlist and the Blocklist and have added the same identifier in both, the Allowlist takes precedence.
 </Callout>
 
-For additional security, adding an individual email address to the Blocklist will also block any attempts to sign up with the email address modified to contain a subaddress.
-
-For example, if you add "john.doe@clerk.dev" as a blocked email address, it means that anybody with "john.doe@clerk.dev" email address will not be able to sign up for your application but in addition "john.doe+anything@clerk.dev" will be blocked as well.
+For additional security, adding an individual email address to the blocklist will also block any attempts to sign up with the email address modified to contain a subaddress. Subaddresses are identified by the presense of any of the following characters in the local part of the email address: `+`, `#`, `=`.
 
-Subaddresses are identified by the presense of any of the following characters in the local part of the email address: `+`, `#`, `=`.
+For example, if you add `john.doe@clerk.dev` as a blocked email address, it means that anybody with `john.doe@clerk.dev` email address will not be able to sign up for your application, including `john.doe+anything@clerk.dev` and any other subaddress.
 
 ## Other restrictions
 
 ### Block email subaddresses
 
-The Block email subaddresses feature allows you to control who can get access to your application. It can restrict email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts.
+The block email subaddresses feature allows you to block all email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts. For example, an email address like `user+sub@clerk.com` will be blocked.
```

Comment:
```suggestion
**Block email subaddresses** allows you to block all email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts. For example, an email address like `user+sub@clerk.com` will be blocked.
```
Since it's a setting in the dashboard, we should bold it for consistency (IMO)